doc/examples/sprite_health.rst

```diff
@@ -8,7 +8,14 @@ Hit Points and Health Bars
 .. image:: sprite_health.png
     :width: 600px
     :align: center
-    :alt: Screenshot of using sprites to shoot things
+    :alt: Screenshot of an enemy shooting at a player with an health indicator bar
+
+This example demonstrates a reasonably efficient way of drawing a health
+bar above a character. The enemy at the center of the screen shoots
```

Comment:
I would add a blank line after the first sentence to separate it from the rest of the paragraph below.